imcsdk/imcmo.py

```diff
@@ -320,7 +321,10 @@ def to_xml(self, xml_doc=None, option=None, elem_name=None, cookie=None):
                             self._dirty_mask & prop.mask != 0)):
                     value = getattr(self, key)
                     if value is not None:
-                        if prop.version <= handle.version:
+                        if handle:
+                            if prop.version <= handle.version:
+                                xml_obj.set(prop.xml_attribute, value)
+                        else:
```

Comment:
as discussed this will still set the property when 
- handle exists
- prop.version > handle.version
Hence the check has been split